### PR TITLE
Update Katex

### PR DESCRIPTION
**Pull request by KarateBrot**
Displaying Latex equations was temporarily broken because of updating to Docusaurus V3 (PR #259)

Docusaurus V3 is dependent on `remark-math 6` and `rehype-katex 7`.
See https://docusaurus.io/docs/markdown-features/math-equations#configuration for more information.

# Before
grafik

# After
grafik

### CODE REVIEW

File: docs/development/mdx-reference.md [hunk 1]
```diff
@@ -181,21 +181,21 @@ graph LR;
 	Quad -- Smart Audio --> VTX;
 ```
 
-## Latex Equations
+## $\LaTeX$ Equations
```
Comment (haslinghuis) [inline docs/development/mdx-reference.md:184]: This does not render right

image

Comment (KarateBrot) [reply docs/development/mdx-reference.md:184]: What is not rendered right? Header 3?

Comment (nerdCopter) [reply docs/development/mdx-reference.md:184]: ??

`aedb7604e` :
image

Comment (nerdCopter) [reply docs/development/mdx-reference.md:184]: [deleted my comment/screenacap, i see it now.]

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**